tutorials/Qubit/Qubit.ipynb

```diff
@@ -0,0 +1,309 @@
+    "        // Since only one qubit is allocated, only its state is printed\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This line changes the qubit from state |0⟩ to state |1⟩\n",
+    "        X(q);\n",
+    "        \n",
+    "        Message(\"State |1⟩:\");\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This line changes the qubit to state |-⟩ = (1/sqrt(2))(|0⟩ - |1⟩)\n",
+    "        // That is, this puts the qubit into a superposition\n",
+    "        // 1/sqrt(2) is approximately 0.707107\n",
+    "        H(q);\n",
+    "        \n",
+    "        Message(\"State |-⟩:\");\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This line changes the qubit to state |-i⟩ = (1/sqrt(2))(|0⟩ - i|1⟩)\n",
+    "        S(q);\n",
+    "        \n",
+    "        Message(\"State |-i⟩:\");\n",
+    "        DumpMachine();\n",
+    "        Message(divider);\n",
+    "        \n",
+    "        // This will put the qubit into an uneven superposition,\n",
+    "        // where the amplitudes of |0⟩ and |1⟩ have different moduli\n",
+    "        Rx(2.0, q);\n",
+    "        \n",
+    "        Message(\"Uneven superposition state:\");\n",
+    "        DumpMachine();\n",
+    "        \n",
+    "        // This line returns the qubit to state |0⟩\n",
+    "        Reset(q);\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%simulate Qubits_Demo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Relative and Global Phase\n",
+    "\n",
+    "You may recall that a complex number has a parameter called its phase. If a complex number $x$ is written in polar form $x = re^{i\\theta}$, its phase is $\\theta$.\n",
+    "\n",
+    "The phase of a basis state is the complex phase of the amplitude of that state. For example, a system in state $\\frac{1 + i}{2}|0\\rangle + \\frac{1 - i}{2}|1\\rangle$, the phase of $|0\\rangle$ is $\\frac{\\pi}{4}$, and the phase of $|1\\rangle$ is $-\\frac{\\pi}{4}$. The difference between these two phases is known as **relative phase**.\n",
+    "\n",
+    "The state of a qubit (or, more generally, the state of a quantum system) is defined up to a global phase. That is, applying a phase to the entire state of a system (multiplying the entire vector by $e^{i\\theta}$ for any real $\\theta$) doesn't actually affect the state of the system.\n",
+    "\n",
+    "$\\alpha|0\\rangle + \\beta|1\\rangle$ is equivalent to $e^{i\\theta}\\big(\\alpha|0\\rangle + \\beta|1\\rangle\\big)$.\n",
+    "\n",
+    "In a situation like that, $\\theta$ is known as the quantum system's **global phase**, **unobservable phase**, or **hidden phase**."
```

Comment:
It would be neat to tie this to Q# with a small demo similar to the previous one. Two things come to mind:
1. Explanations of the phase and the direction columns in DumpMachine output.
2. Some notes on DumpMachine showing the state up to some global phase factor, and with the qubit acquiring -1 phase - usually this can be done by applying some gates and then some more gates that should undo the first ones. This is also one of the frequently asked questions about DumpMachine, so it it a good thing to call out.